sdk/storage/azure-storage-common/vcpkg/vcpkg.json

```diff
@@ -18,7 +18,7 @@
     {
       "name": "azure-core-cpp",
       "default-features": false,
-      "version>=": "1.11.3"
+      "version>=": "1.14.0-beta.1"
```

Comment:
Storage is shipping a GA this release, looking at https://github.com/Azure/azure-sdk-for-cpp/pull/5753.
It can't rely on a beta package, I am assuming we'd wait to merge this after the releases this week.

Reply:
Storage committed their change this morning. I believe we're good to go.